b_cfn_elasticsearch_cloner/cloner_source/index.py

```diff
@@ -13,34 +14,55 @@
 try:
     from dynamodb_decode import DynamodbDecoder
 except ImportError as ex:
-    logger.exception("Failed import.")
+    logger.exception('Failed import.')
     from b_cfn_elasticsearch_cloner.cloner_source.dynamodb_decode import DynamodbDecoder
 
+ES_INDEX = os.environ['ES_INDEX_NAME']
+ES_ENDPOINT = os.environ['ES_DOMAIN_ENDPOINT']
+PRIMARY_KEY_FIELD = os.environ['PRIMARY_KEY_FIELD']
+REGION = os.environ['AWS_REGION']
```

Comment:
Correct me if im wrong, but wouldn't it make sense to get AWS region name from the boto session?